data-otservbr-global/lib/vip/vip_system.lua

```diff
@@ -48,3 +48,29 @@ function Player.onAddVip(self, days)
 	self:setStorageValue(Storage.VipSystem.IsVip, 1)
 end
 
+function checkPremiumAndPrint(player, msgType)
```

Comment:
As it is a global function, it is interesting to capitalize

```suggestion
function CheckPremiumAndPrint(player, msgType)
```